instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/bedrock.py

```diff
@@ -472,6 +499,65 @@ def _on_stream_error_callback(
                 attributes=metrics_attributes,
             )
 
+    def _invoke_agent_on_success(
+        self,
+        span: Span,
+        result: dict,
+        instrumentor_context: _BotocoreInstrumentorContext,
+    ):
+        try:
+            if "completion" in result and isinstance(result["completion"], EventStream):
+                event_stream = result["completion"]
+
+                # Drain the stream so we can instrument AND keep events
+                all_events = list(event_stream)
```

Comment:
Is there a reason we don't want to follow the same approach as we do with `InvokeModelWithResponseStream`?
It looks like we use [a response stream wrapper](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/bedrock.py#L542-L547) that is [an iterator](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/bedrock_utils.py#L166C9-L173).
I wonder what memory implications draining the stream all at once would have?